setup.cfg

```diff
@@ -17,13 +17,15 @@ license_files = LICENSE
 
 # NOTE(lidiz) Adding examples one by one due to pytype aggressive errer:
 # ninja: error: build.ninja:178: multiple rules generate helloworld_pb2.pyi [-w dupbuild=err]
-# TODO(xuanwn): include all files in src/python/grpcio/grpc
+# TODO(sourabhsinghs) : include all packages when merging python/sync/typings to master
 [pytype]
 inputs =
-    src/python/grpcio/grpc/experimental
-    src/python/grpcio/grpc
-    src/python/grpcio_tests/tests_aio
-    src/python/grpcio_observability/grpc_observability
+    src/python/grpcio_admin/grpc_admin
+    src/python/grpcio_channelz/grpc_channelz
+    src/python/grpcio_csds/grpc_csds
+    src/python/grpcio_health_checking/grpc_health
+    src/python/grpcio_reflection/grpc_reflection
+    src/python/grpcio_status/grpc_status
```

Comment:
Same as above, please add a TODO to include observability packages.

Reply:
Already have a TODO for adding back all packages before merge to master